SafeExamBrowser.I18n/Data/id.xml

```diff
@@ -0,0 +1,729 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Text>
+    <Entry key="AboutWindow_LicenseInfo">
+        Aplikasi ini tunduk pada ketentuan Lisensi Publik Mozilla, versi 2.0. Safe Exam Browser menggunakan kerangka kerja dan pustaka pihak ketiga berikut ini:
+    </Entry>
+    <Entry key="AboutWindow_Title">
+        Versi &amp; Informasi Lisensi
+    </Entry>
+    <Entry key="Browser_BlockedContentMessage">
+        Konten diblokir
+    </Entry>
+    <Entry key="Browser_BlockedPageButton">
+        Kembali ke halaman sebelumnya
+    </Entry>
+    <Entry key="Browser_BlockedPageMessage">
+        Akses ke halaman ini tidak diizinkan menurut konfigurasi saat ini.
+    </Entry>
+    <Entry key="Browser_BlockedPageTitle">
+        Halaman Diblokir
+    </Entry>
+    <Entry key="Browser_LoadErrorMessage">
+        Terjadi kesalahan saat memuat halaman "%%URL%%":
+    </Entry>
+    <Entry key="Browser_LoadErrorTitle">
+        Kesalahan Pemuatan Halaman
+    </Entry>
+    <Entry key="Browser_Name">
+        Peramban
+    </Entry>
+    <Entry key="Browser_PrintNotAllowed">
+        Pencetakan tidak diperbolehkan menurut konfigurasi saat ini.
+    </Entry>
+    <Entry key="Browser_Tooltip">
+        Membuka Halaman Web
+    </Entry>
+    <Entry key="BrowserWindow_DeveloperConsoleMenuItem">
+        Konsol Pengembang
+    </Entry>
+    <Entry key="BrowserWindow_Downloading">
+        Mengunduh...
+    </Entry>
+    <Entry key="BrowserWindow_DownloadCancelled">
+        Dibatalkan.
+    </Entry>
+    <Entry key="BrowserWindow_DownloadComplete">
+        Diunduh.
+    </Entry>
+    <Entry key="BrowserWindow_FindCaseSensitive">
+        Pencarian dengan kepekaan kapital
+    </Entry>
+    <Entry key="BrowserWindow_FindMenuItem">
+        Mencari halaman...
+    </Entry>
+    <Entry key="BrowserWindow_ReloadButton">
+        Muat ulang
+    </Entry>
+    <Entry key="BrowserWindow_BackwardButton">
+        Mundur
+    </Entry>
+    <Entry key="BrowserWindow_ForwardButton">
+        Maju
+    </Entry>
+    <Entry key="BrowserWindow_DownloadsButton">
+        Unduh
+    </Entry>
+    <Entry key="BrowserWindow_HomeButton">
+        Beranda
+    </Entry>
+    <Entry key="BrowserWindow_MenuButton">
+        Menu
+    </Entry>
+    <Entry key="BrowserWindow_CloseButton">
+        Tutup
+    </Entry>
+    <Entry key="BrowserWindow_UrlTextBox">
+        Masukkan URL
+    </Entry>
+    <Entry key="BrowserWindow_ZoomLevelReset">
+        Halaman Diperbesar pada %%ZOOM%% %. Klik untuk Mengatur Ulang.
```

Comment:
I don't know which is right `%%ZOOM%% %` or `%%ZOOM%%%`

Reply:
Both are okay, the latter would avoid a space and is the case in the other translations I think (i.e. 75% instead of 75 %).